tools/scripts/generate_sdks.py

```diff
@@ -1,152 +1,190 @@
-﻿# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
this is a wrong script to modify.
You could just delete it.

Reply:
I'm going to do a clean up of all no longer scripts in this directory and merge this back to the right place. 
Moving to draft in the mean time.